src/modules/dashboard/components/CoinMachine/BuyTokens/BuyTokens.tsx

```diff
@@ -230,186 +255,224 @@ const BuyTokens = ({ colony: { colonyAddress }, disabled }: Props) => {
+                                 * Just entering the decimal point will pass it through to EthUsd
+                                 * and that will try to fetch the balance for, which, obviously, will fail
+                                 */
+                                !disabled ? parseFloat(currentSalePrice) : 0
+                              }
+                            />
+                          </div>
+                        )
+                      }
+                    </div>
+                  </div>
+                  <div className={styles.symbols}>
+                    {`${purchaseToken?.symbol}/${sellableToken?.symbol}`}
+                  </div>
                 </div>
-                <div className={styles.amountsValues}>
-                  {!disabled ? (
-                    <div>
-                      {values.amount
-                        ? (
-                            parseInt(values.amount, 10) *
-                            parseFloat(currentSalePrice)
-                          ).toFixed(2)
-                        : ''}
+                <div className={styles.amountsContainer}>
+                  <div className={styles.amounts}>
+                    <div className={styles.amountsLabel}>
+                      <FormattedMessage {...MSG.costLabel} />
                     </div>
-                  ) : (
-                    <div>N/A</div>
-                  )}
-                  {
-                    /*
-                     * @NOTE only show the exchange rate if the token is XDAI/ETH
-                     * since otherwise there's not place we can (currently) fetch
-                     * this data
-                     */
-                    purchaseToken?.address === AddressZero && (
-                      <div>
-                        <EthUsd
-                          appearance={{ theme: 'grey', size: 'small' }}
-                          value={
-                            /*
-                             * @NOTE Set value to 0 if amount is only the decimal point
-                             * Just entering the decimal point will pass it through to EthUsd
-                             * and that will try to fetch the balance for, which, obviously, will fail
-                             */
-                            values.amount
-                              ? parseInt(values.amount, 10) *
+                    <div className={styles.amountsValues}>
+                      {!disabled ? (
+                        <div>
+                          {values.amount
+                            ? (
+                                parseInt(values.amount, 10) *
                                 parseFloat(currentSalePrice)
-                              : '0'
-                          }
-                        />
-                      </div>
-                    )
-                  }
+                              ).toFixed(2)
+                            : ''}
+                        </div>
```

Comment:
Do we need this div if we dont have values.amount?

Reply:
That's from Raul's PR, so I guess we do.